docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,6 +2152,622 @@ Drop all values in mods:part.
+
+.. code-block:: xml
+
+    <relatedItem type="host" displayLabel="Collection">
+      <titleInfo>
+        <title>Botany Department Photographs</title>
+      </titleInfo>
+      <identifier type="local">AR.0488</identifier>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+
+    <https://example.org/objects/1> dbo:collection "Botany Department Photographs, AR.0488" .
+
+`@type='catalog'`'s value, if present, will be represented by the `opaque:sheetmusic_hostItem` property.
+
+`Example record - vanvactor:10012 <https://digital.lib.utk.edu/collections/islandora/object/vanvactor:10012/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <relatedItem type="otherVersion">
+      <titleInfo>
+        <title>Three songs for soprano, alto flute, English horn, and bass clarinet</title>
+      </titleInfo>
+      <identifier type="catalog">M120</identifier>
+    </relatedItem>
+    <relatedItem displayLabel="Project" type="host">
+      <titleInfo>
+        <title>David Van Vactor Music Collection</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Collection" type="host">
+      <titleInfo>
+        <title>David Van Vactor Papers</title>
+      </titleInfo>
+      <identifier>MS.1942</identifier>
+      <location>
+        <url>https://n2t.net/ark:/87290/v8pz5703</url>
+      </location>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> dbo:collection "David Van Vactor Papers, MS.1942" ;
+        dbo:isPartOf <https://n2t.net/ark:/87290/v8pz5703> ;
+        opaque:sheetmusic_hostItem "Three songs for soprano, alto flute, English horn, and bass clarinet, M120" .
+
+`@type='pid'` will not be migrated.
+
+relatedItem/location/url
+------------------------
+Use Case
+^^^^^^^^
+This XPath is used 8516 times, but only has 33 distinct strings.
+
+Justification
+^^^^^^^^^^^^^
+XPath
+^^^^^
+:code:`relatedItem/location/url`
+
+Decision
+^^^^^^^^
+The `dbo:isPartOf` property was selected.
+
+`Example record - ruskin:204 <https://digital.lib.utk.edu/collections/islandora/object/ruskin:204/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <relatedItem displayLabel="Collection" type="host">
+      <titleInfo>
+        <title>Ruskin Cooperative Association Collection</title>
+      </titleInfo>
+      <identifier>MS.0023</identifier>
+      <location>
+        <url>https://n2t.net/ark:/87290/v81g0jf1</url>
+      </location>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+
+    <https://example.org/objects/1> dbo:isPartOf <https://n2t.net/ark:/87290/v81g0jf1> ;
+        dbo:collection "Ruskin Cooperative Association Collection, MS.0023" .
+
+relatedItem/location/physicalLocation/
```

Comment:
Since this category only involves one record, would you like to write a ticket and I'll go ahead and just remove the location/physicalLocation? Let me know if you have previously and I have simply missed it! I think having a whole category for this one troublesome record adds unnecessary complexity to the mapping doc.

Reply:
Done!